src/tests/suite/daos_rebuild_simple.c

```diff
@@ -638,6 +638,49 @@ rebuild_objects(void **state)
 	reintegrate_single_pool_target(arg, ranks_to_kill[0], tgt);
 }
 
+static void
+rebuild_sx_object(void **state)
+{
+	test_arg_t	*arg = *state;
+ 	daos_obj_id_t	oid;
+ 	struct ioreq	req;
+	char		dkey[32];
+ 	const char	akey[] = "test_update akey";
+ 	const char	rec[]  = "test_update record";
+ 	d_rank_t	rank = 2;
+	int		rank_nr = 1;	
+	int		i;
+
+	if (!test_runable(arg, 4))
+		return;
+
+	oid = dts_oid_gen(OC_SX, 0, arg->myrank);
+	ioreq_init(&req, arg->coh, oid, DAOS_IOD_ARRAY, arg);
+	print_message("insert 100 dkeys\n");
+	for (i = 0; i < 100; i++) {
+		sprintf(dkey, "dkey_%d\n", i);
+		insert_single(dkey, akey, 0, (void *)rec, strlen(rec),
+			      DAOS_TX_NONE, &req);
+	}
+ 
+	print_message("insert 100 dkeys done\n");
+	get_killing_rank_by_oid(arg, oid, 1, 0, &rank, &rank_nr);
+ 	/** exclude the target of this obj's replicas */
+ 	daos_exclude_server(arg->pool.pool_uuid, arg->group,
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs